src/python/foglamp/core/storage_server/__main__.py

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env python3
```

Comment:
@amarendra-dianomic  can you please remove this now? and simply call ./storage  from the build directory itself; add TODO for manaul build and cp process (or may be write a shell script and execute that to do that inside build/run/<>

Reply:
Please do not do anything with the build.sh scripts now. Ashwin will look at the build process, let's not propagate two ways of doing things. @ashwinscale can you work with Amarendra here please?